### PR TITLE
RD-6341 Allow absolute paths for backend type imports

### PR DESCRIPTION
**Pull request by kubama**
## Description
Allowed absolute paths for backend type imports

## Screenshots / Videos
N/A

## Checklist

- [X] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [X] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [X] I added proper labels to this PR.

## Tests
CI verification only

 

## Documentation
N/A

### CODE REVIEW

File: app/components/LicensePage.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
-import _ from 'lodash';
```
Comment (Vorbert-Kruk) [inline app/components/LicensePage.tsx:1]: Nice catch 💪

File: app/components/LicensePage.tsx [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-import React, { Component } from 'react';
 import type { ChangeEvent } from 'react';
+import React, { Component } from 'react';
```
Comment (Vorbert-Kruk) [inline app/components/LicensePage.tsx:2]:
```suggestion
import React from 'react';
import type { Component } from 'react';
```

File: app/utils/shared/UrlUtils.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import _ from 'lodash';
-import Const from '../consts';
 import { getUrlWithQueryString } from '../../../backend/sharedUtils';
```
Comment (Vorbert-Kruk) [inline app/utils/shared/UrlUtils.ts:2]: Did we omitted this line or is troublesome to also modify this path? 🤔 
```suggestion
import { getUrlWithQueryString } from 'backend/sharedUtils';
```

**Review by Vorbert-Kruk — COMMENTED** [review]
I did a quick investigation upon searching for possible leftovers and it seems that there are 7 more occurrences of relative `backend` imports (on 1 I've already commented on 😁)

Could we also modify those paths to be absolutes or would it be troublesome? 🤔 

image

**Review by PatrycjaKubiczek — APPROVED** [review]

**Comment by kubama** [comment]
> Could we also modify this path to be absolutes or would it be troublesome? thinking

Good catch, I somehow missed it. Updated.

**Review by Vorbert-Kruk — APPROVED** [review]
LGTM 💪

File: app/components/LicensePage.tsx (hunk 2 above)
Comment (kubama) [reply app/components/LicensePage.tsx:2]: I had to revert this change - `Component` is used as a value so it broke CI.

Comment (Vorbert-Kruk) [reply app/components/LicensePage.tsx:2]: Oh, got it,
Sorry in that case for the incorrect comment

**Review by Vorbert-Kruk — APPROVED** [review]

**Pull request merged**